unstructured/partition/pdf.py

```diff
@@ -475,6 +479,50 @@ def _process_pdfminer_pages(
                         last_modified=metadata_last_modified,
                     )
                     page_elements.append(element)
+        numbered_item = 0
+        updated_page_elements = []  # type: ignore
+        coordinate_system = PixelSpace(width=width, height=height)
+        for page_element in page_elements:
+            if isinstance(page_element, ListItem):
+                numbered_item += 1
```

Comment:
Does "ListItem" cover bulleted lists (non-numerical) as well? 
Is the bulleted broken list case also covered in this change?

Reply:
Yep!